cmd/release/config/config.go

```diff
@@ -78,12 +107,14 @@ type Auth struct {
 
 // Config
 type Config struct {
-	User    *User          `json:"user"`
-	K3s     *K3s           `json:"k3s"`
-	Rancher *Rancher       `json:"rancher"`
-	RKE2    *RKE2          `json:"rke2"`
-	Charts  *ChartsRelease `json:"charts"`
-	Auth    *Auth          `json:"auth"`
+	User      *User          `json:"user"`
+	K3s       *K3s           `json:"k3s"`
+	UI        *UI            `json:"ui"`
```

Comment:
I don't think this code needs to know about the UI repo unless there's code to check that the tag was created from the GitHub action when a tag is created in the dashboard repo.

Reply:
Removed it.